thumbnail_frame.py

```
@@ -1,3 +1,249 @@
-version https://git-lfs.github.com/spec/v1
-oid sha256:a6e8e85c8811fd4e985b313cecb4ac9df343ad2c82894599259f84c2c53c9943
-size 10971
+import tkinter as tk
+from tkinter import ttk
+from PIL import Image, ImageTk
+import sqlite3
+import io
+import time
+from functools import partial
```

Ruff

`functools.partial` imported but unused

Remove unused import: `functools.partial`

(F401)